mturk/mturk.ipynb

```diff
@@ -687,31 +696,70 @@
     "pass"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create Compensation HIT\n",
```

Comment:
Let's move this up a few cells to be under "Add Custom Quals" and above "Download Data". It's nice to have the data download at the end so you can write a few custom cells to make sure your data looks ok once it is downloaded.